compiler/rustc_ast/src/ast.rs

```diff
@@ -3160,9 +3160,13 @@ mod size_asserts {
     static_assert_size!(Impl, 136);
     static_assert_size!(Item, 136);
     static_assert_size!(ItemKind, 64);
-    static_assert_size!(LitKind, 24);
+    // This can be removed after i128:128 is in the bootstrap compiler's target.
+    #[cfg(not(bootstrap))]
+    static_assert_size!(LitKind, 32);
```

Comment:
This failed for me on macos x64 with llvmorg-18-init-9505-g10664813 from chromium . Expected was 32, but actual was still 24

Reply:
https://github.com/rust-lang/rust/pull/116672/files#diff-00567f9be31678eec46651e6157f0eaaa47d1f344eb61b8a63f87edfa7bf6b14R3169 also failed and was still 40 instead of 48

Reply:
hmmm... seeing the error on the asserts mod.rs as well. Maybe I'm doing something wrong...

Reply:
Not sure if you have seen but there is some more discussion about this pattern in https://github.com/rust-lang/rust/pull/116672#discussion_r1357340257

Reply:
Also, feel free to drop into the Zulip discussion since you are taking a look at this https://rust-lang.zulipchat.com/#narrow/stream/187780-t-compiler.2Fwg-llvm/topic/LLVM.20alignment.20of.20i128

Reply:
yes, but in this case I'm building with a version of llvm 18 that has the alignment change so the new value should be correct, right?

Reply:
I see what I did wrong here, I removed `#[cfg(not(bootstrap))]` because I thought it was no longer needed since I was building against llvm 18, but on further inspection it looks like the problem is a mismatch between the bootstrap llvm version and the output rustc llvm version

Reply:
I guess that isn't quite right, but @maurer explained the problem and now I understand what's going on